Makefile

```diff
@@ -179,6 +179,13 @@ deploy-with-olm: deploy-olm load-on-kind build-and-push-bundle-images ## deploys
 	$(KUSTOMIZE) build config/olm-install | kubectl apply -f -
 	VERSION=$(CSV_VERSION) NAMESPACE=$(NAMESPACE) hack/wait-for-csv.sh
 
+deploy-olm-real: export OPERATOR_SDK=operator-sdk
+deploy-olm-real: deploy-olm build-and-push-bundle-images ## deploys the operator via OLM on a real cluster
```

Comment:
We should remove the `deploy-olm` dependency?

Also, I wonder if we can refactor this a bit to avoid duplications with deploy-with-olm